aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -14,7 +14,7 @@
 
 apiVersion: v1
 name: kubernetes-dashboard
-version: 2.3.0
+version: 2.3.1
```

Comment:
Could you bump minor version instead of patch? This is a new feature according to semver.

Reply:
Sure thing. Bumped to 2.4.0